src/QMCWaveFunctions/BsplineFactory/BsplineReaderBase.h

```diff
@@ -179,6 +181,9 @@ struct BsplineReaderBase
   /** Set the checkNorm variable */
   inline void setCheckNorm(bool new_checknorm) { checkNorm = new_checknorm; };
 
+  /** Set the orbital rotation flag. Rotations are applied to balance the real/imaginary components. */
+  inline void setRotate(bool new_rotate) { rotate = new_rotate; };
```

Comment:
keep a note for my self. We will need to kill all the setXX function and pass via constructor.